sequel/benchmarks/bm_sequel_validations_invalid.rb

```diff
@@ -0,0 +1,43 @@
+require 'bundler/setup'
+require 'sequel'
+require_relative 'support/benchmark_rails'
+
+DB = Sequel.connect(ENV.fetch('DATABASE_URL'))
+
+DB.create_table(:posts) do
+  primary_key :id
+  column :title, "varchar(255)"
+  column :author, "varchar(255)"
+  column :body, "Text"
+  column :sequence, 'integer'
+  column :age, 'integer'
+  column :legacy_code, "varchar(255)"
+  column :size, "varchar(255)"
```

Comment:
> Columns are generally created by specifying the column type as the method name
Can you update all the columns to use the column type as the name instead? Thanks! :)